commands/visualstudio.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var Q = require('q');
+
+var lib = require('manifoldjs-lib');
+
+var CustomError = lib.CustomError,
+    exec = lib.processTools.exec,
+    fileTools = lib.fileTools,
+    log = lib.log;
+
+var open = require('./open');
+
+function isWindows10Version (version) {
+  return /^10/.test(version);
+}
+
+function getWindowsVersion (callback) {

```

Comment:
You're right. The visualstudio command is actually deprecated and was replaced by the open command. At one point, it was removed altogether but then we resuscitated the code from the original codebase and changed it a little bit, but it probably needs more work, or just die :).